src/utils/fxa.js

```diff
@@ -155,6 +155,29 @@ async function sendMetricsFlowPing (path) {
 }
 /* c8 ignore stop */
 
+/**
+ * @param {string} subscriptionId 
+ */
+// TODO: Add unit test when changing this code:
```

Comment:
I don't think it's realistic that we'll ever write unit tests for it - they wouldn't be very useful either, I don't think :)

```suggestion
// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
```

Reply:
I think you changed this on a different line (though I guess it applies there too) :)